website/docs/getting-started/installation.mdx

```diff
@@ -20,42 +20,97 @@ import TOCInline from '@theme/TOCInline';
 <TOCInline toc={toc} />
 ```
 
-## Option A: Compiling Locally (Recommended)
+## Option A: Download Pre-Built Releases (Recommended)
 
-### Installing Dependencies
+### a) Pre-Built Binary
```

Comment:
It's a bit strange that "Option A" has sub-headings "a)" and "b)". Same for "Option B". Maybe just enumerate headings as "1." and "1.1."?

Reply:
I followed the existing documentation which was using A and B, so a) and b) were natural choices for subsections. But I also found this odd. Do you want to drop the "Option" prefix altogether or just "Option A"->"Option 1" etc.?

Reply:
Ideally, we make clear that these sections are all alternatives, not a sequence of steps to be done. Maybe prefix everything with "Option"? And have a sub-numbering?